packages/server/src/sdk/app/links/links.ts

```diff
@@ -0,0 +1,32 @@
+import { context } from "@budibase/backend-core"
+import { isTableId } from "@budibase/backend-core/src/docIds"
+import { LinkDocument, LinkDocumentValue } from "@budibase/types"
+import { ViewName, getQueryIndex } from "../../../../src/db/utils"
+
+export async function fetch(tableId: string): Promise<LinkDocumentValue[]> {
+  if (!isTableId(tableId)) {
+    throw new Error(`Invalid tableId: ${tableId}`)
+  }
+
+  const db = context.getAppDB()
+  const params: any = { startkey: [tableId], endkey: [tableId, {}] }
```

Comment:
This should be typable with the `DatabaseQueryOpts` type

Reply:
I did initially try that but `startkey` and `endkey` are both expected to be strings, not arrays. I'm uncertain what their string values should be, or how this works in the place I copied it from.

Reply:
Ah - I would update them in that case - *usually* they are strings, but realistically it could be a number, or an array of string or number type.